common/locales/en/settings.json

```diff
@@ -41,7 +41,7 @@
     "json": "(JSON)",
     "customDayStart": "Custom Day Start",
     "24HrClock": "24Hr Clock",
-    "clockInfo": "HabitRPG defaults to check and reset your Dailies at midnight each day. You can customize that here (Enter number between 0 and 24).",
+    "customDayStartInfo": "HabitRPG defaults to checking and resetting your Dailies at midnight each day. You can customize that here. Enter a number between 0 and 24. It is recommended that you first read <a href=\"http://habitrpg.wikia.com/wiki/Custom_Day_Start\" target=\"_blank\">this advice about how to change it</a>.",
```

Comment:
@Alys when you need a string to be updated immediately, ping me on Slack and I'll run the script to update them before deploy